### ISSUE TITLE
Where to change the tag focus indicator position?

### ISSUE THREAD

**Issue by linuxdotexe** [HikariKnight/material-awesome#12]
The tag focus indicator position is to the left by defualt and I want it up or down since I moved the tag-list to the top panel but I cannot find it. Where can I do this customization?

**Comment by HikariKnight** [comment]
This i am a bit unsure about myself. but my guess is that it would be in either the tag-list module/widget files or inside the theme files.

maybe @PapyElGringo could share some of his knowledge if he happens to remember where that setting is specifically :)

**Comment by linuxdotexe** [comment]
there is an option to adjust the thickness of the indicator but not the position. so i changed the thickness thing so the whole tag is highlighted. now i removed the tag display entirely since it resized with the wibar and i can't see the icons anymore. i want to remove icons entirely and replace it with text (like in [DistroTube's Xmonad](https://imgur.com/eLjx5AZ)), with the existing functionality. please check if you can help me with that. thanks for the reply.

**Comment by HikariKnight** [comment]
look into the tag-list.lua that is what builds the tags and add the icons should in theory be possible to add text there i believe :)
Closing this as the issue is for the awesome4.3-stable build

**Comment by linuxdotexe** [comment]
well i tried all that... didn't work that's why i came to the mentioned forks. just look into it if you can... what i identified is that having a text tag is simple but the code for adding icons is different. i made a backup and deleted the code that gives it icons and added the tags code from the original rc.lua. it removed the tag display, and added the workspaces to the previous ones i.e., i had 5 workspaces with icons and i removed them and added this code from rc.lua:
```
    awful.tag({ "1", "2", "3", "4", "5", "6", "7", "8", "9" }, s, awful.layout.layouts[1])
```
now what happened was the original 5 remained and the new 9 were added to the 5 and none had display. i kinda went nuts over that and removed the entire tags display. i don't know much lua so i couldn't do any thing else and i didn't get the documentation too. look into it if you can. thanks in advance :)

**Comment by HikariKnight** [comment]
i dont know any lua myself, most of the changes i have done so far is guesswork from the awesomewm api and edits to what papyelgringo and manilarome  has done in their versions 😃

but i am slowly learning more as i need to change things for my own personal preference and i am currently trying to add back the indicators to material awesome 2.0 (which is in the master branch now)

i found out that the indicators might be done using gradients in the default-theme.lua, however i tried porting over papys theme to 2.0 but the results were not close to the old indicators. so for now the indicators are scrapped.

I might message papy directly and ask how they did the indicators if i really miss them enough (although they are more easily seen than a highlight and more appropriate to the material guidelines)

**Comment by linuxdotexe** [comment]
great! [here](https://github.com/sainivasmangu/gruver) is my gruvified version of the original haha(i am like really new to git and github so i didn't fork it but i cloned and then uploaded.). lemme know if you like it :)
cheers!
